internal/attestation/attestation.go

```diff
@@ -35,7 +36,7 @@ var (
 type Attestation[T any] interface {
 	Data() []byte
 	Statement() T
-	Signatures() []signature.EntitySignature
+	Output() output.Attestation
```

Comment:
This looks odd, attestation containing attestation?

Reply:
That's not what this means. It's a method, not an attribute. An `attestation.Attestation` knows how to represent itself as an `output.Attestation`. This is similar to how you'd use a `String` method to get a string representation of a struct.